src/Withdrawal.java

```diff
@@ -117,7 +117,10 @@ private int displayMenuOfAmounts()
             case 3: // corresponding amount from amounts array
             case 4:
             case 5:
-               userChoice = amounts[ input ]; // save user's choice
+               // userChoice = amounts[ input ]; // save user's choice
+               screen.displayMessage("\nHow much do you want to withdraw? ");
+               int customAmount = keypad.getInput();
```

Comment:
Do we need to add a check on the type of the inserted input to guard against possible runtime exception?

Reply:
Attualmente i nostri terminali accettano solamente numeri. Non c'èe la possibilità di inserire caratteri.
Potrebbe valere la pena verificare anche con il PM se possa essere una funzionalità per il futuro.

Ho creato un ticket (SPIKE) per tracciare questa analisi.